### ISSUE TITLE
does not work on a SuSe Leap 15.5 installation

### ISSUE THREAD

**Issue by Blaesen** [yuezk/GlobalProtect-openconnect#365]
Hello,

Unfortunately the client does not work on a SuSe Leap 15.5 installation. The connection is established, but an outgoing RDP connection does not work. Kernel 5.14 is installed. I'm happy to pay for the client if it works.

The free Global Protect Client under Manjaro Linux 24, Kernel 6.6 works perfectly, but yours under suse doesn't. In both cases I use Remmina + FreeRDP for the RDP connection.

What can I do? You can reach me by email at blaesen@gmx.net.

greeting
Uwe

**Comment by yuezk** [comment]
Yes. I found this problem in my VM 15.5 as well. I will investigate deeper later.
